frontend/src/utilities/useDraggableTableControlled.ts

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+
+export type Row<T> = T & {
+  id: string;
+};
+
+type useDraggableTableControlled<T> = {
```

Comment:
```suggestion
type useDraggableTableControlledProps<T> = {
```

Reply:
Technically not the props, but the return type of the hook.

I did drop the capitalization accidentally. 

Looking at the other hooks in the utilities, it could  be `DraggableTableControlled`?

Reply:
Yeah, `DraggableTableControlled` seems appropriate